docs/main/user-guides/local-storage-user-guide/local-pv-zfs/zfs-configuration.md

```diff
@@ -226,54 +226,54 @@ zfspv-pool/pvc-34133838-0d0d-11ea-96e3-42010a800114    96K  4.00G    96K  legacy
 
 ## Parameters
 
```

Comment:
Let's not change the case of all the below parameters, may break the functionality